docs/configuration.md

```diff
@@ -80,10 +80,13 @@ Property
 **envoy-control.envoy.snapshot.incoming-permissions.tls-authentication.protocol.cipher-suites**                                                    | A list of cipher suites to use                                                                                                                                                                 | list of "ECDHE-ECDSA-AES128-GCM-SHA256", "ECDHE-RSA-AES128-GCM-SHA256"
 **envoy-control.envoy.snapshot.incoming-permissions.tls-authentication.mtls-enabled-tag**                                                          | Name of tag that is used to identify if mTLS should be enabled for cluster (note that `envoy-control.envoy.snapshot.egress.http2.enabled` should also be enabled)                              | mtls:enabled
 **envoy-control.envoy.snapshot.incoming-permissions.tls-authentication.san-uri-format**                                                            | URI format for SAN field that will be matched in client/server validation. {service-name} will be replaced by service name from discovery source.                                              | spiffe://{service-name}
+**envoy-control.envoy.snapshot.incoming-permissions.tls-authentication.san-uri-wildcard-regex**                                                    | URI format for SAN field for services that use wildcard clients. This means that every client that presents with a valid certificate and his client name matches this regex will be allowed.   | spiffe://.+
```

Comment:
Sounds great, I did it this way because I was thinking about that there might be a reason to have a "catch-all" regex like: `.+` without any prefix but if you don't see the need to do this, that approach is more safe.